src/components/DayOverview/DayOverview.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import supabase, { Schema } from "@/supabase";
+import DayOverviewButton from "@/components/DayOverview/DayOverviewButton";
+
+export type ParcelOfSpecificDateLocation = {
+    collection_datetime: Schema["parcels"]["collection_datetime"];
+    clients: {
+        flagged_for_attention: Schema["clients"]["flagged_for_attention"];
```

Comment:
I disagree with this, in that I don't think there will be changes in the schema, and I think it would be better readability wise to have the types explicitly written out. See what others think tho.